### PR TITLE
Remove "Projects" section from docs

### LINKED ISSUE #932: Rename `projects` section in docs
It is basically a section about configuration, we should maybe remove it and put it in `Getting started`?
It is also incomplete on Forge's side

### PR DESCRIPTION
**Pull request by ATREAY**
Closes #932

## Introduced changes

The project configuration section has been relocated beneath the "Getting Started" category, and the Forge section has been finalized within this context.

## Breaking changes

-Nil-

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [ ] Added relevant tests
- [x] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/projects/configuration.md [hunk 1]
```diff
@@ -16,6 +16,42 @@ exit_first = true
...
+
+Forge makes things easier by letting you set up fork configurations inside the Scarb.toml file. Instead of repeatedly typing out the same details on the command line, you can define them in the file. 
+
+For example, you can set up different forks like this:
+
+```toml
+[[tool.snforge.fork]]
+name = "ethereum_mainnet"
+url = "http://your.rpc.url"
+block_id.tag = "Latest"
```
Comment (piotmag769) [inline docs/src/projects/configuration.md:29]: I think it is enough to add this to the example above tbh. Please change the name though the node is a Starknet one, not Ethereum

**Comment by ATREAY** [comment]
hey @piotmag769,  I've implemented the changes you recommended. Could you review them and let me know if you need any more adjustments?

File: docs/src/projects/configuration.md [hunk 2]
```diff
@@ -16,6 +16,21 @@ exit_first = true
...
+
+```toml
+[[tool.snforge.fork]]
+name = "Starknet"
+url = "http://your.rpc.url"
+block_id.tag = "Latest"
+```
+
+To learn more, check out the page about [Fork Testing](../testing/fork-testing.md).
+
```
Comment (cptartur) [inline docs/src/projects/configuration.md:33]: I'm not sure if it makes sense to duplicate this information here. That's one more place to maintain it and I don't think anyone looking for information on how to configure forking would search in this file.

There's already info about that here https://github.com/foundry-rs/starknet-foundry/blob/0ea77905ea437a70b2861cd3476a7e8654c23927/docs/src/testing/fork-testing.md?plain=1#L45


```suggestion
```

**Comment by ATREAY** [comment]
Hey @cptartur, I've made the necessary changes. Could you please take a look whenever you have a moment?

**Review by piotmag769 — COMMENTED** [review]
I think it's fine, it's on me to not describe it clearly enough. I think we should just expand the 
```toml
# ...
[tool.snforge]
exit_first = true
# ...
```
to sth like 
```toml
# ...
[tool.snforge]
exit_first = true
fuzzer_runs = 1234
fuzzer_seed = 1111

[[tool.snforge.fork]]
name = "SOME_NAME"
url = "http://your.third.rpc.url"
block_id.number = "123"
# ...
```

Also the sentences before that should be changed to 
`It is possible to configure some of Forge's behavior through Scarb.toml.`

**Comment by cptartur** [comment]
> I think it's fine, it's on me to not describe it clearly enough. I think we should just expand the
> 
> ```toml
> # ...
> [tool.snforge]
> exit_first = true
> # ...
> ```
> 
> to sth like
> 
> ```toml
> # ...
> [tool.snforge]
> exit_first = true
> fuzzer_runs = 1234
> fuzzer_seed = 1111
> 
> [[tool.snforge.fork]]
> name = "SOME_NAME"
> url = "http://your.third.rpc.url"
> block_id.number = "123"
> # ...
> ```
> 
> Also the sentences before that should be changed to `It is possible to configure some of Forge's behavior through Scarb.toml.`

I think we can merge this one and create a separate issue for the fork configs.

**Review by cptartur — APPROVED** [review]

**Pull request merged**